tests/run-docker

```diff
@@ -14,10 +14,9 @@ if [ "$CMD" == "reset" ]; then
     make reset
     exit 0
 elif [ "$CMD" == "build" ]; then
-    make build
+    make build-plugins
 fi
 
-make up "$@"
-
-echo "Use 'docker-compose logs <service>' to display logs"
+make env
```

Comment:
it should be done line 11, else `make reset` will fail if the .env does not exists

Reply:
Fixed